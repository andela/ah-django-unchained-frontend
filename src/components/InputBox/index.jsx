import React from 'react';

export const InputBox = props => {
  return (
    <input
      type={props.type}
      className={props.className}
      name={props.name}
      id={props.id}
      placeholder={props.placeholder}
    />
  );
};

export default InputBox;
