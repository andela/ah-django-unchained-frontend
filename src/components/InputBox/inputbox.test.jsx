import React from "react";
import { shallow } from "enzyme";
import InputBox from "./";

describe("InputBox", () => {
  it("should be defined", () => {
    expect(InputBox).toBeDefined();
  });
  it("should render correctly", () => {
    const tree = shallow(
        <InputBox
        type="password"
        name="password"
        id="password"
        placeholder="password"
        className="col-sm-6"
      />
    );
    expect(tree).toMatchSnapshot();
  });
});
