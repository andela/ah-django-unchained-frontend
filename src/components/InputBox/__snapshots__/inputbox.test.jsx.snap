// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputBox should render correctly 1`] = `
<input
  className="col-sm-6"
  id="password"
  name="password"
  placeholder="password"
  type="password"
/>
`;
